examples/v1/pipelineruns/alpha/pipelinerun-with-pipelines-in-pipelines.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: tekton.dev/v1
```

Comment:
The example cannot be part of the CI since this will result in a failure without the feature implemented. If you would like to maintain it in the repo, please add it in the doc itself. We used to maintain `no-ci` examples but that has changed now.: